src/stratis_cli/_actions/_logical.py

```diff
@@ -158,6 +161,12 @@ def filesystem_used(props):
                 props, "Used", lambda x: str(Range(x)), TABLE_FAILURE_STRING
             )
 
+        def format_uuid(mofilesystem_uuid):
```

Comment:
I'ld just make the decision once, in the specification of the function rather than every time the function is run:

```
format_uuid = <the identity function> if namespace.unhyphenated_uuids else lambda uuid: <the transforming expression>
```

This uses ```lambda``` keyword, which is Python's way of defining anonymous functions and also its if-expression: ```<true-expr> if <bool condition> else <false-expr>```.